### PR TITLE
Updated info about `exonum-app` [ECR-3074]

### PR DESCRIPTION
**Pull request by skletsun**
- $PATH should contain path to `installed_app_path/bin' folder
- removed all info about $EXONUM_HOME
- mentioned that $JAVA_HOME can be used to configure JVM used by the app

### CODE REVIEW

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -35,34 +35,16 @@ Follow these steps:
...
-    2>&1 > /dev/null |\
-    grep 'java.home' |\
-    awk '{print $3}')}"
-  LIBJVM_PATH="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
-
-  export LD_LIBRARY_PATH="${LIBJVM_PATH}"
-  ```
-
-  <!-- cspell:enable -->
+- By default, the application tries to determine the location of JVM from the already installed Java, but you can configure the JVM to use with the JAVA_HOME environment variable.
```
Comment (vitvakatu) [inline src/get-started/java-binding.md]:
```suggestion
- By default, the application tries to determine the location of JVM from the already
  installed Java, but you can configure the JVM to use with the `JAVA_HOME`
  environment variable.
```

**Comment by dmitry-timofeev** [comment]
I think EXONUM_HOME shall be kept in the section on testing (currently it mentions EXONUM_**JAVA**:

> `$EXONUM_JAVA` environment variable should point at [add: Exonum Java] installation location, ~~as specified in How to Run a Service section~~ [no longer specified there].

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -35,34 +35,18 @@ Follow these steps:
...
-    2>&1 > /dev/null |\
-    grep 'java.home' |\
-    awk '{print $3}')}"
-  LIBJVM_PATH="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
-
-  export LD_LIBRARY_PATH="${LIBJVM_PATH}"
-  ```
-
-  <!-- cspell:enable -->
+- By default, the application tries to determine the location of JVM from the already
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Updated ECR-3048 to move this to configuration section once we have it.

I think the description is not accurate: _by default_ (or, with higher precedence), Exonum uses the JVM located in `JAVA_HOME`, if it is set. If it is not set, it uses the JVM on the `PATH`.

**Comment by dmitry-timofeev** [comment]
It must probably be closed as obsolete.

**Pull request closed without merging**